src/Raven.Server/Rachis/Elector.cs

```diff
@@ -27,6 +36,20 @@ public void HandleVoteRequest()
                     {
                         var rv = _connection.Read<RequestVote>(context);
 
+                        if (rv.ElectionResult == ElectionResult.Won)
```

Comment:
You aren't checking the _term_ here. We might be skipping ahead of ourselves here, no?
Or are you relying on the check being done in the follower? If so, we need an explicit comment.